base_layer/wallet/Cargo.toml

```diff
@@ -34,9 +34,10 @@ futures =  { version = "=0.3.0-alpha.19", package = "futures-preview", features
 tokio = "0.2.0-alpha.4"
 tower = "0.3.0-alpha.2"
 tokio-executor = { version ="^0.2.0-alpha.6", features = ["threadpool"] }
+simple_logger = "1.3.0"
```

Comment:
We shouldn't need logger implementations in our libraries.

env_logger is already in dev_deps; rather use that for tests.

Reply:
Oh, this should come out. Needed it for debugging a swift app because you can't set the env_logger path variables in the iOS sim.